Windows/windows/ingest/parser.yml

```diff
@@ -358,6 +359,49 @@ stages:
             {% for component in parsed_message_xml.result.AuditBase.ContextComponents.Component   %}{%if "ProxyServer" in component%}{{component["ProxyServer"].strip()}}{% endif %}{% endfor %}
         filter: "{{json.event.SourceName == 'AD FS Auditing'}}"
 
+      - set:
+          action.properties.AdditionalActionsID: "{{ json.event.get('Additional Actions ID') }}"
+          action.properties.AdditionalActionsString: "{{ json.event.get('Additional Actions String') }}"
+
+      - set:
+          action.properties.CertIssuerName: "{{json_event.message.CertIssuerName}}"
+          action.properties.CertSerialNumber: "{{json_event.message.CertSerialNumber}}"
+          action.properties.CertThumbprint: "{{json_event.message.CertThumbprint}}"
```

Comment:
If the destination field as the same name that the source field, it should be automatically handled by 

https://github.com/SEKOIA-IO/intake-formats/blob/7df70e5ce335285ece986e0639fbdf4984dc7d51/Windows/windows/ingest/parser.yml#L298

you only need to declare the field in fields.yml